guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/GitMetadataStore.java

```diff
@@ -37,6 +37,14 @@ void write( String name,
                 String origin );
 
     /**
+     * Writes the meta information about a repository.
+     * @param name The name of the repository in organizationalUnit/repositoryName format
+     * @param metadata The metadata object that stores information about repository
+     */
+    void write( String name,
```

Comment:
@manstis it's necessary to have flexbility on repository name, as we have flexible naming structure. user's can have just "reponame" or "ou/reponame"... and OU is not exactly mandatory... as it can be just "username/reponame".